.github/lock.yml

```diff
@@ -0,0 +1,41 @@
+# Configuration for Lock Threads - https://github.com/dessant/lock-threads
+
+# Number of days of inactivity before a closed issue or pull request is locked
+daysUntilLock: 10
+
+# Skip issues and pull requests created before a given timestamp. Timestamp must
+# follow ISO 8601 (`YYYY-MM-DD`). Set to `false` to disable
+skipCreatedBefore: false
+
+# Issues and pull requests with these labels will be ignored. Set to `[]` to disable
+exemptLabels:
```

Comment:
Shouldn't we add the next labels: 
discussion, enhancement, help wanted, more-information-needed, bug

Reply:
Why? If the issue is closed we can lock it.